src/main/kotlin/no/nav/syfo/melding/api/PaminnelseRequestDTO.kt

```diff
@@ -0,0 +1,28 @@
+package no.nav.syfo.melding.api
+
+import no.nav.syfo.melding.domain.*
+import java.time.OffsetDateTime
+import java.util.*
+
+data class PaminnelseRequestDTO(
+    val document: List<DocumentComponentDTO>,
+)
+
+fun PaminnelseRequestDTO.toMeldingTilBehandler(opprinneligMelding: MeldingTilBehandler): MeldingTilBehandler {
+    val now = OffsetDateTime.now()
+    return MeldingTilBehandler(
+        uuid = UUID.randomUUID(),
+        createdAt = now,
+        type = MeldingType.FORESPORSEL_PASIENT_PAMINNELSE,
+        conversationRef = opprinneligMelding.conversationRef, // TODO: Stemmer dette?
```

Comment:
Blir det riktig å sette disse til `conversationRef` for meldingen man purrer på?

Reply:
Ja, jeg ville tenkt at den skal kobles på den originale meldingen i samme conversation-tråd👍🏼

Reply:
Ja, tror det blir riktig. Men parentRef bør muligens peke til den opprinnelige meldingen?